sklearn/neighbors/tests/test_nca.py

```diff
@@ -0,0 +1,511 @@
+import pytest
```

Comment:
I would add here a header with license and authors on top of the file like we do in other places

Reply:
That's right it's missing, but did you mean to say in the main file `nca.py` ? I looked in two or three tests code and it didn't seem they had an author section/licence